app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -1,48 +1,63 @@
 import React from "react";
 
 import { Formik, Form as FormikForm } from "formik";
-import { Button, Pane } from "neetoui";
+import { Button, Pane, Toastr } from "neetoui";
 import { Input, Textarea } from "neetoui/formik";
+import { useTranslation } from "react-i18next";
 
 import { NOTES_FORM_VALIDATION_SCHEMA } from "../constants";
 
-const Form = ({ onClose, note, isEdit }) => (
-  <Formik
-    initialValues={note}
-    validationSchema={NOTES_FORM_VALIDATION_SCHEMA}
-    onSubmit={onClose}
-  >
-    {({ isSubmitting }) => (
-      <FormikForm className="w-full">
-        <Pane.Body className="space-y-6">
-          <Input
-            required
-            className="w-full flex-grow-0"
-            label="Title"
-            name="title"
-          />
-          <Textarea
-            required
-            className="w-full flex-grow-0"
-            label="Description"
-            name="description"
-            rows={8}
-          />
-        </Pane.Body>
-        <Pane.Footer>
-          <Button
-            className="mr-3"
-            disabled={isSubmitting}
-            label={isEdit ? "Update" : "Save changes"}
-            loading={isSubmitting}
-            style="primary"
-            type="submit"
-          />
-          <Button label="Cancel" style="text" onClick={onClose} />
-        </Pane.Footer>
-      </FormikForm>
-    )}
-  </Formik>
-);
+const Form = ({ onClose, note, setNotes }) => {
+  const { t } = useTranslation();
+
+  const handleSubmit = async values => {
+    try {
+      setNotes(notes => [...notes, values]);
+      Toastr.success(t("notes.noteCreateSuccess"));
+      onClose();
+    } catch (err) {
+      logger.error(err);
+    }
+  };
```

Comment:
Why's this an `async` block? What `async` operation is being performed here?

Reply:
@yedhink no async operation here, just the code from wheel and forgot to remove it in this PR, have removed in the upcoming PR.

Reply:
Ok. Got it. Thanks for clarifying the same.